### ISSUE TITLE
Kotlin Bootcamp issue: 

### ISSUE THREAD

**Issue by JDLowe7** [google-developer-training/kotlin-bootcamp#13]
**Describe the problem**
Poor wording/ conflating mutability with fixed length

**In which step of the codelab can this issue be found?**
Task 5, Step 2: Create arrays

**Versions**
What version of IntelliJ IDEA are you using?

**Additional information**
"Unlike lists in Kotlin, which have mutable and immutable versions, there is no mutable version of an Array. Once you create an array, the size is fixed. You can't add or remove elements, except by copying to a new array."

Maybe the writer meant no immutable version. You can change an element of an array, so it isn't immutable.  The following sentence seems to imply that a fixed size(can't add or remove elements) is what immutable means. This might confuse new programmers; even if they are not the intended audience I think it would be good to clarify.

tl;dr
immutable ->
Add something like "Also", "Additionally", or "Furthermore" before "Once you create an array" to disassociate the concept of mutability with whether or not something has a fixed-sized

**codelab:** kotlin-bootcamp
